scripts/delete.rb

```diff
@@ -0,0 +1 @@
+Person.where(title: 'Queen of Meereen and Mother of Dragons').destroy
```

Comment:
`.where` returns an array of results, even if there is only one match. Try using `find_by` to only return one result, or use `destroy_all` to destroy everything in the array